recipes-core/initrdscripts/tegra-minimal-init/platform-preboot.sh

```diff
@@ -35,8 +35,22 @@ fi
 if [ -z "$rootdev" ]; then
     echo "[FAIL]"
     if [ -n "$mayberoot" ]; then
-	rootdev="$mayberoot"
+        if [ "`echo $mayberoot | cut -c1-5`" = "UUID=" ]; then
```

Comment:
I'm fine with trying to support these additional ways to specify the rootfs device, but will the `/dev/disk/by-xxxx` symlinks be present?  I thought it was udev (or systemd-udevd) that created those, and that wouldn't be running in a minimal initramfs.

Using `lsblk` to look up partitions by the specified attributes might work better, maybe?

Reply:
@madisongh thanks for the input, you are absolutely right, will change that in V2.